taxonomies/ingredients.txt

```diff
@@ -33829,6 +33836,18 @@ wikipedia:en:https://en.wikipedia.org/wiki/Orange_juice
 wikidata:en:Q219059
 # orange-juice 1401 products in 9 languages @2018-09-28
 
+<en:orange
+en:blood orange
+fr:orange sanguine
+es:naranja sanguina
+it:arancia rossa
+de:Blutorange
+wikidata:en:https://www.wikidata.org/wiki/Q303669
```

Comment:
```suggestion
wikidata:en:Q303669
```

Reply:
github doesn't allow suggestions to be merged for the ingredients.txt file (too big I guess). I'll merge and we can fix after.